README.md

```diff
@@ -103,6 +104,30 @@ $ fdiff [OPTIONS] [PRE-FONT FILE URL] [POST-FONT FILE FILE PATH]
 
 ⭐ **Tip**: Remote git repository hosting services (like Github) support access to files on different git branches by URL.  Use these repository branch URL to compare fonts across git branches in your repository.
 
+#### As Git's diff driver for fonts
+
+Git can be configured to automatically use a specific tool to diff specific file types. These are the steps to use `fdiff` as the default diff output for font files.
+
+1. Tell Git how to run the tool. As with most git configuration options this may be set for a repository (`--local`), for your user (`--global`), or system (`--system`). We recommend setting this at the user level. Assuming `fdiff` is available in your path, this setting should do the trick:
+
+        git config --global diff.fdiff.command 'fdiff -c --git'
+
+    This will write it to your `$HOME/.gitconfig` file looking like this:
+
+    ```gitconfig
+    [diff "fdiff"]
+        command = fdiff -c --git
+    ```
+
+    Of course you may also edit it there.
+
+2. Tell Git to use that specific tool for supported file types. This may also be done at multiple places. Each repository may have it's own `.gitattributes` file, a user may have one setting global defaults in `$XDG_HOME/git/attributes`, or there may be a system wide default file. Wherever you choose to place this, the lines are the same:
```

Comment:
I confirmed that this approach works and can reproduce the fontTools exception issue with a local `.gitattributes` file in the root of the repository.

I needed to modify the configuration to glob the extensions though:

```
*.otf   diff=fdiff
*.ttf   diff=fdiff
```

Without that change, I don't see `fdiff` output.

Reply:
Edited in https://github.com/source-foundry/fdiff/pull/48/commits/b2370df96f01d0d854339f3adbea69dbe5e927be